### DISCUSSION TITLE
PyCharm license issue

### DISCUSSION THREAD

**Discussion by shelsonby** [community/community#56403] [GitHub Education]
### Hi everyone,

I get approved for the student developer pack by GitHub but when I try to get Pycharm license with my GitHub student developer pack I cannot do it? I guess py pack's features has not been approved yet? 

Can you help guys to make this process faster. I need Pycharm..

**Comment by Ntshangase** [comment]
Hi @shelsonby 

It's possible for GitHub to approve your student developer application but take some time (usually up to 72 hours) for you to be fully accredited so that you can enjoy the student benefits.

I would suggest that you try applying to pyCharm after 72hours.

**Comment by samus-aran** [comment]
Hey @shelsonby  👋

We've updated our application and re-verification process to ensure that GitHub and our partners can offer exceptional, free benefits to students worldwide during their academic journey.

With this update, when you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away. Your educational benefits will arrive within 72 hours of this verification. However, during busy periods like the start of a new semester, it may take a bit longer.

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!
